templates/maas/features.html

```diff
@@ -0,0 +1,526 @@
+          <li>Test your hardware</li>
+          <li>Assign to physical <a href="/maas/docs/how-to-manage-machine-groups#p-19384-manage-availability-zones">zones</a></li>
+        </ul>',
+        "cta_html": '<a href="/maas/docs/how-to-manage-machines"
+          onclick="dataLayer.push({&quot;event&quot;: &quot;GAEvent&quot;, &quot;eventCategory&quot;: &quot;Feature&quot;, &quot;eventAction&quot;: &quot;Click&quot;, &quot;eventLabel&quot;: &quot;Fast deployment&quot; });"
+          aria-label="Link to machine lifecycle page on MAAS documentation">Learn more about the node actions&nbsp;&rsaquo;</a>'
+      },
+      {
+        "title_text": 'Provision with any <abbr title="Operating system">OS</abbr> image',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/e2e4cbf4-provision_with_1.jpg",
+          description="operating system image page",
+        ),
+        "description_html": '<p>
+          Select the OS you require in your data centre
+        </p>
+        <ul>
+          <li>Ubuntu</li>
+          <li>CentOS</li>
+          <li>Windows</li>
+          <li>RHEL</li>
+        </ul>
+        <p>Import, update or sync the images or connect to an onsite mirror to work offline.</p>',
+        "cta_html": '<a href="/maas/docs/how-to-manage-images"
+          onclick="dataLayer.push({&quot;event&quot;: &quot;GAEvent&quot;, &quot;eventCategory&quot;: &quot;Feature&quot;, &quot;eventAction&quot;: &quot;Click&quot;, &quot;eventLabel&quot;: &quot;Learn more about images&quot; });"
+          aria-label="Link to the install config images page on MAAS documentation">Learn more about images&nbsp;&rsaquo;</a>'
+      },
+      {
+        "title_text": 'Deploy with the press of a button',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/3b72e050-deploy_with_1.jpg",
+          description="node deployment",
+        ),
+        "description_html": "<ol>
+            <li>Select the machine</li>
+            <li>Choose the OS and architecture</li>
+            <li>Press 'deploy'</li>
+          </ol>",
+        "cta_html": '<a href="/maas/docs/how-to-manage-machines#p-9078-deploy-machines"
+          onclick="dataLayer.push({&quot;event&quot;: &quot;GAEvent&quot;, &quot;eventCategory&quot;: &quot;Feature&quot;, &quot;eventAction&quot;: &quot;Click&quot;, &quot;eventLabel&quot;: &quot;Fast deployment&quot; });"
+          aria-label="Link to the install and config page of node deployment on MAAS documentation">Learn more about
+          deployment&nbsp;&rsaquo;</a>'
+      },
+      {
+        "title_text": 'Get all the information you need',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/4e398f57-get_all_the_1.jpg",
+          description="commissioning list",
+        ),
+        "description_html": "<ul>
+          <li>
+            Dig into your hardware details that MAAS gathers during commissioning including; processors, memory, storage,
```

Comment:
non-blocking since it's written exactly this way in the [copydoc](https://docs.google.com/document/d/1KcYwNK0pJyiDVEDq4vTpv6BX_rpSDnxUxtAvAnsRKyw/edit?disco=AAABsh-WfWw) - maybe this should be a colon instead of a semicolon. I let the writer know.